stagecoach/jobs/publishers/examples/publish_all.py

```diff
@@ -40,7 +41,7 @@ async def publish_all(assets):
                 CoinbaseFetcher,
                 CryptowatchFetcher,
```

Comment:
delete please

Reply:
We should no longer use the cryptowach fetcher, the data is bad

Reply:
Ok